drivers/console/multi_console.c

```diff
@@ -10,14 +10,16 @@
 
 #include <drivers/console.h>
 
+#include "./console_private.h"
+
 console_t *console_list;
 uint8_t console_state = CONSOLE_FLAG_BOOT;
 
+IMPORT_SYM(console_t *, __STACKS_START__, stacks_start)
+IMPORT_SYM(console_t *, __STACKS_END__, stacks_end)
```

Comment:
Why does this need to move?

Reply:
The `IMPORT_SYM` macro declares an extern variable, which generates a nested-extern warning when used inside a function.
The way this macro is used in every other file in the project is outside of any function.

Reply:
Okay, fair enough.